evm/scripts/start-sequencer.sh

```diff
@@ -50,9 +50,10 @@ cp app.toml /root/.world-evm/config/app.toml
 # CometBFT API
 sed -i'.bak' 's#"tcp://127.0.0.1:26657"#"tcp://0.0.0.0:26657"#g' /root/.world-evm/config/config.toml
 
-# Faucet setup. changes the default account to the provided account, then supplies it with tokens.
-sed -i'.bak' "s#'20f33ce90a13a4b5e7697e3544c3083b8f8a51d4'#'$FAUCET_ADDR'#g" /root/.world-evm/config/genesis.json
```

Comment:
for some reason this env variable injection into the sed command wasn't working. not sure how to fix. so just gonna hard code it with the default for now.

all this address does is give account EVM funds. this way you can test cross-shard messaging.